apps/docs/src/app/platform/component-docs/platform-forms/checkbox/platform-checkbox-examples/platform-checkbox-styling.component.ts

```diff
@@ -0,0 +1,30 @@
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { Component, DoCheck } from '@angular/core';
+
+@Component({
+    selector: 'fdp-checkbox-style',
+    templateUrl: 'platform-checkbox-styling.component.html',
+})
+export class PlatformChekboxStyleComponent implements DoCheck {
+    values = { trueValue: 'Yes', falseValue: 'No', thirdStateValue: 'I dont have an opinion' };
```

Comment:
I dont think on application level anybody will work with CV like this. please see my comment for the component ts file

Reply:
implementation is changed now. now only checked value is expected from user.